src/Keboola/StorageApi/Client.php

```diff
@@ -126,6 +131,12 @@ public function __construct(array $config = array())
             $this->maxJobPollWaitPeriodSeconds = (int) $config['maxJobPollWaitPeriodSeconds'];
         }
 
+        if (isset($config['retryDelay'])) {
+            $this->retryDelay = $config['retryDelay'];
+        } else {
+            $this->retryDelay = self::getDefaultJobPollDelay($this->maxJobPollWaitPeriodSeconds);
```

Comment:
`maxJobPollWaitPeriodSeconds` parameter is used only if `retryDelay` is not provided, which is a bit confusing.
At least  `maxJobPollWaitPeriodSeconds` shouldn't be a class property.

Reply:
We can throw exception if both parameters will be provided. Or refactor it somehow to accept only `retryDelay` optional parameter?